🌐 Update German translation for `docs/de/docs/python-types.md`

nilslindemann opened the pull request:
Based on #3048.

← `features.md` (#10284)
→ `async.md` (#10449)

[German translation progress](https://github.com/tiangolo/fastapi/discussions/10582)

Commit:
German translation of docs/python-types.md

* Full translation
* Based on #3048
* In sync with current en/docs/python-types.md
* Hand-translated (Google translate rarely used)
* Checked for errors using LanguageTool
* Manually checked

tiangolo commented:
📝 Docs preview for commit afe69470676bd4ade7f3456e8a8c76c0875b749a at: https://9c6245a7.fastapitiangolo.pages.dev

DevSpace88 reviewed (changes requested):
Good translation, just a few points here and there.

nilslindemann commented on docs/de/docs/python-types.md:
```diff
@@ -0,0 +1,537 @@
+Aber selbst wenn Sie **FastAPI** nie verwenden, wird es für Sie nützlich sein, ein wenig darüber zu lernen.
+
+!!! Hinweis
+    Wenn Sie ein Python-Experte sind und bereits alles über Typ-Hinweise wissen, springen Sie zum nächsten Kapitel.
```
Stimme ich nicht mit überein. Finde "Springen zu" besser.

DevSpace88 replied:
Perfekt

nilslindemann replied:
Ach was, ich lasse das so, das ist verständlich genug.

Commit:
Vorschläge von DevSpace88 übernommen

tiangolo commented:
📝 Docs preview for commit 9ea7e308331ead4fb6b0e91d1a6c7964d1c499b3 at: https://d60f35f6.fastapitiangolo.pages.dev

Commit:
Weiteren Vorschlag von DevSpace88 übernommen

nilslindemann commented:
Danke, @DevSpace88, fürs Feedback.

Commit:
Weniger eindeutschen

Die englischen Worte sind in IT-Kreisen verbreiteter.

Siehe die Diskussion bei #10290

tiangolo commented:
📝 Docs preview for commit 9692b239fbef3435e027729cc1d906946441e3b5 at: https://d1a91d81.fastapitiangolo.pages.dev

Commit:
Standardwert -> Defaultwert

Passt besser. Steht auch bei Leo als erste Übersetzung

tiangolo commented:
📝 Docs preview for commit 76394f48db922cc8ade38d2b7829d1f02ecd91dc at: https://9ea630f4.fastapitiangolo.pages.dev

Commit:
Weniger eindeutschen

Die englischen Worte sind in IT-Kreisen verbreiteter.

Siehe die Diskussion bei #10290

tiangolo commented:
📝 Docs preview for commit c86695d24b1615bd8fbee98e0cc059971077487f at: https://dfa97ccb.fastapitiangolo.pages.dev

Commit:
Falsch ersetzten Link durch korrekten ersetzen

tiangolo commented:
📝 Docs preview for commit 4b214711b975b74917fe7d831b820f1eb80844dc at: https://3f5e72c2.fastapitiangolo.pages.dev

Commit:
Fix formatting

Translating the text changed the icon and coloring of the box

tiangolo commented:
📝 Docs preview for commit 980e6ee553c7159c0bb98b6b6b840567ae434f1e at: https://a670357d.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/python-types.md

tiangolo commented:
📝 Docs preview for commit 1b5e8b5882193e66c425c9dec640f7b5b868fe68 at: https://2ea203e3.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/python-types.md

tiangolo commented:
📝 Docs preview for commit 18ef532b5cf66526b61253b9d52142b3eb5a7a3e at: https://1016c583.fastapitiangolo.pages.dev

Commit:
Improve writing of some words

Fits https://github.com/tiangolo/fastapi/discussions/10369

tiangolo commented:
📝 Docs preview for commit b6c23161af3cb49871043ad4760b9604ad16aa53 at: https://ffe6f943.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/python-types.md

tiangolo commented:
📝 Docs preview for commit 386f25caf3fd2155c26537d1eedea9e9de126f88 at: https://61ea9074.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/python-types.md

tiangolo commented:
📝 Docs preview for commit 1cba6f5e87c45f82fb5c693784988274ae13d7ff at: https://e09cba59.fastapitiangolo.pages.dev

Commit:
Adapt to #10442

tiangolo commented:
📝 Docs preview for commit 848ec20dae50fbaf4dfd1ecdb0424d66d6a1f43a at: https://e95fa85f.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/python-types.md

tiangolo commented:
📝 Docs preview for commit 83fc242c4cb837c2b242f094160eeedc9beea72e at: https://ac9eb3c3.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/python-types.md

tiangolo commented:
📝 Docs preview for commit 9e5c505d7f30330506aa6d9982592d3939dc4505 at: https://8450aff6.fastapitiangolo.pages.dev

Commit:
"Tip" -> "Tipp"

tiangolo commented:
📝 Docs preview for commit b0c64a2acfaf98dc36980a17e86808c04344e7e4 at: https://0b1b46d8.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/python-types.md

Commit:
Use typographic quotes

tiangolo commented:
📝 Docs preview for commit 645d867a3b9c17530a79e4a057fa00ab4747853b at: https://ea54955a.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/python-types.md

tiangolo commented:
📝 Docs preview for commit 7c79e6d14dae78d20564e44a9eb2927494259618 at: https://7fdbd89d.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/python-types.md

tiangolo commented:
📝 Docs preview for commit a19316fd04c2165973e93cc84d4715bbac141dd6 at: https://e249f7ff.fastapitiangolo.pages.dev

Commit:
"Standard " -> "Standard-"

Commit:
"werfen" -> "auslösen"

tiangolo commented:
📝 Docs preview for commit e82b2af4c9ac234712c270535dd7674b53f40911 at: https://876ea139.fastapitiangolo.pages.dev

Commit:
"Python " -> "Python-", etc.

tiangolo commented:
📝 Docs preview for commit 4426b769d881deccff2da6a9fd3bd169f5c29ddb at: https://8a636762.fastapitiangolo.pages.dev

Commit:
"das gleiche" vs "dasselbe"

tiangolo commented:
📝 Docs preview for commit 41614e307c364bfd0061f8fe6c169e07283ccba2 at: https://2beb5c2f.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/python-types.md

tiangolo commented:
📝 Docs preview for commit 456da88b12135b9deb337c0a01170b571b13c56a at: https://082d3276.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/python-types.md

tiangolo commented:
📝 Docs preview for commit 0b3a9e77d0d1ce8acb630dda21979a6c8b17494e at: https://b277766e.fastapitiangolo.pages.dev

Commit:
"z.B." -> "z. B."

Commit:
Remove "-"

tiangolo commented:
📝 Docs preview for commit 44cbe3832dc1c613251139969e7797a6c936a299 at: https://b99d287e.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/python-types.md

tiangolo commented:
📝 Docs preview for commit 22d285a2638b75c79a16f1f1b5bc62c4b1649be9 at: https://b7e536ac.fastapitiangolo.pages.dev

tiangolo commented:
Awesome @nilslindemann! :rocket: 

As you are the champion of German translations and you have done so many, and with so much attention to detail, I'm gonna go ahead and merge this. :tada: 

If others see things that could be improved later, we can check that in new PRs. :nerd_face:
